benchmarks/benchmark_serving.py

```diff
@@ -735,6 +748,12 @@ def main(args: argparse.Namespace):
       default="/tmp/request-outputs.json",
       help="File path to store request outputs",
   )
+  parser.add_argument(
+      "--run-eval",
```

Comment:
Can you add this change to readme?

Reply:
pylint check failed, could you fix the check error?

Reply:
Done.